internal/protocol/params.go

```diff
@@ -2,6 +2,9 @@ package protocol
 
 import "time"
 
+// DesiredReceiveBufferSize is the kernel UDP receive buffer size that we'd like to use.
+const DesiredReceiveBufferSize = (1 << 20) * 2 // 2 MB
```

Comment:
Not sure if this is what we really need, we might need to adjust this constant after more extensive benchmarking.